src/Kmd.Logic.Identity.Authorization/TokenResponse.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+
+namespace Kmd.Logic.Identity.Authorization
+{
+    internal class TokenResponse
+    {
+        [JsonProperty("token_type")]
+        public string TokenType { get; set; }
+
+        [JsonProperty("expires_in")]
+        public int ExpiresIn { get; set; }
+
+        [JsonProperty("ext_expires_in")]
```

Comment:
Not sure we really need this (see https://stackoverflow.com/questions/45681890/oauth-with-azure-ad-v2-0-what-is-the-ext-expires-in-parameter-returned-by-azure).  If the underlying token issuer is AAD then it will be in the returned token, but if it's B2C then it wont be.  Either way, my understanding is that it's a fallback mechanism in case the token issuer can't be contacted. It would require the bearer token validation in the API to support it as a fallback for expires_in.  Given we are moving towards B2C being the primary issuer perhaps we can remove it? I haven't thought about this deeply, just starting a convo for now :)